.gitattributes

```diff
@@ -0,0 +1 @@
+* -text
```

Comment:
I think this is unfortunate naming on Git's part.
The `text` attribute only controls line ending conversion. It doesn't affect whether Git treats the file as text or binary.

See:
https://git-scm.com/docs/gitattributes#_text
https://stackoverflow.com/a/10017566/5916310